community/modules/compute/gke-node-pool/variables.tf

```diff
@@ -134,16 +134,16 @@ variable "compact_placement" {
   default     = false
 }
 
-variable "service_account" {
-  description = "Service account to use with the system node pool"
-  type = object({
-    email  = string,
-    scopes = set(string)
-  })
-  default = {
-    email  = null
-    scopes = ["https://www.googleapis.com/auth/cloud-platform"]
-  }
+variable "service_account_email" {
+  description = "Service account e-mail address to use with the node pool"
+  type        = string
+  default     = null
+}
+
+variable "scopes" {
```

Comment:
I think my only concern here is that my instinct would be to name it "service_account_scopes" in part to ensure that they always land in the documentation right next to one another (and in email/scopes order). WDYT?

Reply:
SGTM and done.